### PR TITLE
Add support for generating CTRL-BREAK

### PR DESCRIPTION
**Pull request by kazarmy**
In some situations CTRL-C is blocked and so it would be useful to have the ability to generate CTRL-BREAK.

### CODE REVIEW

**Comment by zenflow** [comment]
@kazarmy Cool! Didn't really think much about the ctrl-break event, but I guess it could be useful too.

Not sure but maybe, since it exists, the `dwProcessGroupId` argument (which judging by [docs](https://docs.microsoft.com/en-us/windows/console/generateconsolectrlevent) applies to ctrl-break events but not ctrl-c) could be useful too. I did some refactoring of your changes to allow passing this argument to generateCtrlBreak/generateCtrlBreakAsync. It defaults to `0` as before.

Also fixed the demo script (can now be run by `node demo generateCtrlBreak 0`) and updated the readme & package metadata.

If there's nothing else to do that you can think of, please approve the PR and I will squash merge and make a semver-minor release.

**Review by zenflow — APPROVED** [review]

**Comment by kazarmy** [comment]
> If there's nothing else to do that you can think of, please approve the PR and I will squash merge and make a semver-minor release.

Well there could be an actual TypeScript `index.d.ts` file for type definitions, but that can be in a separate PR (or be part of DefinitelyTyped), and I don't think the package name needs to change :blush:

File: generate-ctrl-event.ps1 [hunk 1]
```diff
@@ -13,4 +13,4 @@ public static class Kernel32
 }
 "@
 
-[Kernel32]::GenerateConsoleCtrlEvent(0, 0)
+[Kernel32]::GenerateConsoleCtrlEvent($args[0], 0)
```
Comment (kazarmy) [inline generate-ctrl-event.ps1]: Shouldn't you also obtain `dwProcessGroupId` from `$args`?

Comment (zenflow) [reply generate-ctrl-event.ps1]: Wow I swear I made that change.. thanks for taking a look! Fixed in 401a7bec62761c701ec2e2fc11c1271d49642c54

**Pull request merged**